sql/logic_test.go

```diff
@@ -201,28 +201,27 @@ func (t logicTest) run(path string) {
 			// blank line or a line of the form "xx values hashing to yyy". The
 			// latter format is used by sqllogictest when a large number of results
 			// match the query.
-			if !s.Scan() {
-				t.Fatalf("%d: expected query results line\n", s.line)
```

Comment:
This was only occurring at the end of a file. Without the fatal we'll determine the query has no results which seems reasonable.